storage/Exception.cc

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (c) [2014-2015] Novell, Inc.
```

Comment:
Yes. Copied from the other files in that directory. It may very well be that we need to re-import the legalese there for all files to be up to date with all the company reorganizations going on over the years. But we should do that consistently for all files IMHO, not just for individual ones.